package.json

```diff
@@ -74,6 +74,24 @@
 					"scope": "application",
 					"default": true
 				},
+				"remote.SSH.preferredAuthenticationMethods": {
+					"type": "array",
+					"items": {
+						"type": "string"
+					},
+					"description": "List of preferred authentication methods for SSH connections.",
+					"scope": "application",
+					"default": [
+						"publickey",
+						"password",
+						"hostbased",
+						"keyboard-interactive",
+						"gssapi-with-mic",
+						"gssapi-keyex",
+						"gssapi",
+						"none"
+					]
+				},
```

Comment:
I don't think exposing a setting is needed, parsing ssh config properly and using the same default logic as openssh in case config is not present should be enough

Reply:
I've changed it to remove the settings from package.json. I also changed it so that the list is only those that we currently support (`password`, `publickey`, `keyboard-interactive`).